Tools/CollectSystemInfo.ps1

```diff
@@ -0,0 +1,197 @@
+param (
+    [switch]$IncludeSensitiveData
+)
+
+function Show-Warning {
+    param (
+        [string]$Message
+    )
+    Write-Warning $Message
+}
+
+function Get-SystemConfiguration {
+    Write-Host "Collecting system configuration..."
+    try {
+        Start-Process -FilePath "msinfo32.exe" -ArgumentList "/report", "msinfo32.txt" -Wait

```

Comment:
Why do you need `Start-Process` in this case but don't use it when starting `wevtutil` or `reg` for example?

Reply:
because it is a graphical application